inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/ExtractNodeUtils.java

```diff
@@ -385,7 +385,6 @@ public static MongoExtractNode createExtractNode(MongoDBSource source) {
                 fieldInfos,
                 null,
                 properties,
-                source.getPrimaryKey(),
```

Comment:
If the primary key cannot be changed, can we remove this field in the manager module?

Reply:
Yes, you can.

Reply:
OK, I'll remove them later.